pontoon/checks/libraries/comparelocales.py

```diff
@@ -0,0 +1,166 @@
+from collections import (
+    namedtuple,
+)
+
+from compare_locales.checks import getChecker
+from compare_locales.parser import (
+    FluentParser,
+)
+
+CommentEntity = namedtuple(
+    'Comment', (
+        'all',
+    )
+)
+
+File = namedtuple(
+    'File', (
+        'file',
+        'locale',
+    )
+)
+
+# Because we can't pass the context to all entities passed to compare locales,
+# we have to create our equivalents of compare-locale's internal classes.
+ComparePropertiesEntity = namedtuple(
+    'ComparePropertiesEntity',
+    (
+        'key',
+        'val',
+        'raw_val',
```

Comment:
Silme doesn't seem to modify val on write, but compare-locales modifies `raw_val` to `val` in https://hg.mozilla.org/l10n/compare-locales/file/cdd7bc8c4232/compare_locales/parser.py#l436.

Reply:
@mathjazz is it okay to copy that function from `compare-locales` and apply here?